egui/src/introspection.rs

```diff
@@ -142,6 +142,7 @@ impl Widget for &mut epaint::TessellationOptions {
                 debug_paint_clip_rects,
                 debug_paint_text_rects,
                 debug_ignore_clip_rects,
+                bezier_flattern_tolerence:_,
```

Comment:
Add a slider for it so we can test how different values affect the fidelity of the line!

Reply:
added a slider to demonstrate the tolerance which has ranged from 0.0001 to 10.0.

Reply:
I meant: add a slider here. This is the place where the user can tweak all the options

Reply:
got your point, I added it as part of new PR #1178